ClavaWeaver/src/pt/up/fe/specs/clava/weaver/CxxWeaver.java

```diff
@@ -2079,7 +2079,7 @@ public List<LaraResourceProvider> getNpmResources() {
         return Stream.concat(
                 super.getNpmResources().stream(),
                 Arrays.asList(ClavaApiJsResource.values()).stream())
-            .toList();
+            .collect(Collectors.toList());
```

Comment:
Revert, .toList() is more recent

Reply:
At the time, we were targeting Java 14. Now that Java 17 is the default we can use that method directly.

Resolved in bbb65de65